src/content/docs/logs/ui-data/query-syntax-logs.mdx

```diff
@@ -410,31 +411,108 @@ The following operators can only be used by numeric attributes:
   </tbody>
 </table>
 
-## Example: Search string in URI for deployments [#example-deployments]
-
-You can build a New Relic URL that includes the log search string in the URI. This is useful, for example, when you want to include a direct link to the New Relic **Logs** UI in a Kubernetes deployment. You want the link to pass a custom URL populated with the search for a specific container name.
-
-Here is an example URL:
-
-```
-https://one.newrelic.com/launcher/logger.log-launcher?platform[accountId]=1234567&launcher=jyJpc0VudGl0&pane=zyJuZXJkbGV0SWQiO=
-```
-
-In this search, the query is your Lucene query. Edit these values as applicable:
-
-* `accountId` is your account.
-* `eventTypes` are your log partitions (usually `Log`).
-
-You do not need to edit the other values.
-
-The `launcher` is a Base64 encoding of:
-
-```
-'{"isEntitled":true,"query":"foo:\\"bar\\" environment:\\"production\\"","eventTypes":["Log"]}'
-```
-
-The `pane` is a Base64 encoding of:
-
-```
-'{"nerdletId":"logger.log-tailer","accountId":1234567}'
-```
+## Logs query examples [#query]
+
+If you have Apache logs, we recommend using our built in parsing rules for [Apache logs](/docs/logs/ui-data/built-log-parsing-rules/#apache) to parse the logs into attributes. To use the built-in parsing, simply add the `logtype: apache` attribute to your logging.yml configuration on the hosts for your Apache logs.  
+
+### Example: Query Apache logs
+
+<Callout variant="tip" title="About Apache logs">
+If you have Apache logs, we recommend using our built in parsing rules for [Apache logs](/docs/logs/ui-data/built-log-parsing-rules/#apache) to parse the logs into attributes. To use the built-in parsing, simply add the `logtype: apache` attribute to your logging.yml configuration on the hosts for your Apache logs.
```

Comment:
This note has the same content as the above paragraph. I'd highlight logging.yml somehow, maybe `logging.yml`